webapp/packages/core-theming/src/styles/main/color.pure.scss

```diff
@@ -6,8 +6,8 @@
  * you may not use this file except in compliance with the License.
  */
 
-:disabled, [aria-disabled="true"] {
+:disabled,
+[aria-disabled='true'] {
+  cursor: text;
   pointer-events: none;
-  cursor: default;
-  opacity: 0.5;
```

Comment:
this attributes can be on any element, we sure that we want to have 'text' cursor on each? (for example buttons, menu items)